### PR TITLE
Add content to Principle of Minimum Disclosure section.

### PR DESCRIPTION
**Pull request by msporny**
Please review. This PR addresses issue https://github.com/opencreds/vc-data-model/issues/7.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1238,10 +1238,33 @@
       <section>
         <h3>Atomize Information for Principle of Least Authority</h3>
 
-        <p class="issue">
-When issuing information, issue it in the smallest pieces possible.
-When requesting information, only ask for the data points that you need.
-Issue abstract claims alongside actual data when possible.
+        <p>
+Privacy issues arise when issuers bundle too much information into a
+claim set. For example, a drivers license containing a persons ID number,
```
Comment (dlongley) [inline index.html]: `drivers` => `driver's` and `person` => `person's`

File: index.html [hunk 2]
```diff
@@ -1238,10 +1238,33 @@
...
+height, weight, birthday, and home address is an example of a claim set
+containing far too much information if one of the intended usages of the
```
Comment (dlongley) [inline index.html]: `far too much...` => `more information than is necessary to establish that the person is above a certain age.`

File: index.html [hunk 3]
```diff
@@ -1238,10 +1238,33 @@
...
+license is to establish that the person is above a certain age.
+        </p>
+
+        <p>
+It is considered a best practice for issuers to atomize information when
+possible and issue multiple different claim sets. For example, an issuer
```
Comment (dlongley) [inline index.html]: To `atomize information` or `use a signature scheme that allows for selective disclosure`

File: index.html [hunk 4]
```diff
@@ -1238,10 +1238,33 @@
...
+that issues drivers licenses could issue a claim set containing every
```
Comment (dlongley) [inline index.html]: `drivers` => `driver's`

File: index.html [hunk 5]
```diff
@@ -1238,10 +1238,33 @@
       <section>
         <h3>Atomize Information for Principle of Least Authority</h3>
```
Comment (dlongley) [inline index.html]: This should be `Principle of Minimum Disclosure` not `Least Authority`.

File: index.html [hunk 6]
```diff
@@ -1238,10 +1238,33 @@
...
+height, weight, birthday, and home address is an example of a claim set
+containing far too much information if one of the intended usages of the
+license is to establish that the person is above a certain age.
+        </p>
+
+        <p>
+It is considered a best practice for issuers to atomize information when
+possible and issue multiple different claim sets. For example, an issuer
+that issues drivers licenses could issue a claim set containing every
+attribute that appears on a drivers license in addition to individual
```
Comment (dlongley) [inline index.html]: `driver` => `driver's`

File: index.html [hunk 7]
```diff
@@ -1238,10 +1238,33 @@
...
+claims (a singular claim containing the persons birthday), and individual
```
Comment (dlongley) [inline index.html]: `persons` => `person's`

File: index.html [hunk 8]
```diff
@@ -1238,10 +1238,33 @@
...
+claims that are more abstract (a singular claim containing an
+<code>ageOver</code> attribute). In addition, the issuer is encouraged to
+provide HTTP endpoints for retrieving single-use bearer claims to promote
```
Comment (dlongley) [inline index.html]: `HTTP endpoints` => `secure HTTP endpoints`

File: index.html (hunk 5 above)
Comment (msporny) [inline index.html]: Fixed.

File: index.html (hunk 1 above)
Comment (msporny) [inline index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (msporny) [inline index.html]: Fixed.

File: index.html (hunk 3 above)
Comment (msporny) [inline index.html]: Fixed.

File: index.html (hunk 4 above)
Comment (msporny) [inline index.html]: Fixed.

File: index.html (hunk 7 above)
Comment (msporny) [inline index.html]: Fixed.

File: index.html (hunk 8 above)
Comment (msporny) [inline index.html]: Fixed.

**Comment by msporny** [comment]
@amigus, @jandrieu - need reviews from you guys on this PR.

**Comment by msporny** [comment]
@amigus, @jandrieu second ping for review on this PR.

**Comment by msporny** [comment]
@amigus, @jandrieu third ping for review on this PR.

**Comment by jandrieu** [comment]
The opening section could be a bit more rigorous and connect to regulatory requirements.

For example, instead of "Privacy issues arise when issuers bundle too much information into a
claim set. " 

Consider 
"Privacy violations occur when information divulged in one context leaks into another. Accepted best practice for preventing such violations is to limit the information requested, and received, to the absolute minimum necessary. This minimal disclosure approach is required by regulation in multiple jurisdictions, including HIPAA in the US and GDPR in the EU."

"With verifiable claims, minimal disclosure for issuers means limiting the content of a claim to the minimum required by potential inspectors for expected use. For inspectors, it means limiting the scope of claims request or required for accessing services."

Then continue as before:

"For example, a driver's license containing a driver's ID number, height, weight, birthday, and home address is an example of a claim set containing more information than is necessary to establish that the person is above a certain age."

**Pull request merged**